_layouts/default.html

```diff
@@ -13,14 +13,19 @@
 
     {{ content }}
 
-    <footer>
+    <footer class='buffer-footer'>
       <div class="row">
         <div class="large-12 columns">
-          <a href="https://github.com/emacs-berlin/emacs-berlin.org" target="_blank">Site source code is available on Github.</a>
+          <a href="https://github.com/emacs-berlin/emacs-berlin.org" target="_blank">Site source code is available on GitHub</a>
         </div>
       </div>
     </footer>
 
+    <footer class='mode-line'>
+      U:--- {{page.filename}}&lt;emacs-berlin.org&gt;&nbsp;&nbsp;&nbsp;Git:gh-pages&nbsp;&nbsp;(Markdown Jekyll)
```

Comment:
The template language is very forgiving - when the variable is not defined, then this will simply be empty.

This is funny though: apparenlty `page.path` returns exactly what is needed here, no modifications necessary. I pushed a change that uses it instead.